common/docker/client.go

```diff
@@ -10,6 +10,10 @@ type ContainerObserver interface {
 	ContainerDied(ident string)
 }
 
+type ContainerObserver2 interface {
```

Comment:
yuck!

How much hassle is it to add the method to`ContainerObserver` and change all the implementations?